### PR TITLE
CONTRIB-7962: Handle missing language strings for new recording formats

### PR DESCRIPTION
**Pull request by laurentdavid**
On the page that will display the recording list, sometimes the recording type is unknown (because of a new type), so we cannot find the language string.
Aim here is to provide a way to "gracefully degrade", by displaying a capitalised version of the recording type (i.e. notes will become "Notes"....).

I took the initiative (we can roll it back) to change the language strings so it displays "Presentation" instead of "presentation" in the label as the end results seems more consistent when we don't find a couple of strings. See screenshots provided on the ticket (https://tracker.moodle.org/browse/CONTRIB-7962).

Unit tests added for the new function.

### CODE REVIEW

File: lang/en/bigbluebuttonbn.php [hunk 1]
```diff
@@ -351,11 +351,11 @@
...
-$string['view_recording_format_podcast'] = 'podcast';
-$string['view_recording_format_presentation'] = 'presentation';
-$string['view_recording_format_screenshare'] = 'screenshare';
-$string['view_recording_format_statistics'] = 'statistics';
-$string['view_recording_format_video'] = 'video';
+$string['view_recording_format_podcast'] = 'Podcast';
+$string['view_recording_format_presentation'] = 'Presentation';
+$string['view_recording_format_screenshare'] = 'Screenshare';
+$string['view_recording_format_statistics'] = 'Statistics';
+$string['view_recording_format_video'] = 'Video';
```
Comment (jfederico) [inline lang/en/bigbluebuttonbn.php]: We are still missing notes=Notes. The fallback will do it normally, but no for non-English sites

File: locallib.php [hunk 2]
```diff
@@ -1656,6 +1656,24 @@ function bigbluebuttonbn_get_recording_data_row_type($recording, $bbbsession, $p
...
 
+/**
+ * Helper function to handle yet unknown recording types
+ *
+ * @param string $playbacktype : for now presentation, video, statistics, capture, notes, podcast
+ *
+ * @return either the matching language string or a capitalised version of the provided string
+ */
+function bigbluebuttonbn_get_recording_type_text($playbacktype) {
+    // Check first if string exists
```
Comment (jfederico) [inline locallib.php]: If you run the codechecker you should see an error here. Comments must end in a dot .

Comment (laurentdavid) [reply locallib.php]: Ok, that's right. Just amended and shortened the comment.

File: lang/en/bigbluebuttonbn.php (hunk 1 above)
Comment (laurentdavid) [reply lang/en/bigbluebuttonbn.php]: Added it to the language string.

**Pull request merged**